packages/block-editor/src/components/rich-text/get-rich-text-values.js

```diff
@@ -81,7 +83,7 @@ function addValuesForElements( children, ...args ) {
 function addValuesForBlocks( values, blocks ) {
 	for ( let i = 0; i < blocks.length; i++ ) {
 		const { name, attributes, innerBlocks } = blocks[ i ];
-		const saveElement = getSaveElement( name, attributes );
+		const saveElement = getSaveElement( name, attributes, innerBlocks );
```

Comment:
We should avoid passing this down as block because they'll be serialised, which is unperformant.

Reply:
I created an alternative PR: #52682